aws/aws-explanation/ec2-image-retention-policy.rst

```diff
@@ -0,0 +1,60 @@
+AWS image retention policies
+=============================
+
+Ubuntu images on AWS have three stage of life cycle : *release* > *deprecation* > *deletion*. 
+
+Whenever a new image is built and released to AWS, all the previous serials for that image are deprecated except for the last 3. A deprecated image is not visible on the AWS console, but it can still be found and launched from the `AWS CLI`_ using its AMI ID. You can also view the image details using the AWS CLI `describe images`_ command by including the ``--include-deprecated`` flag.
```

Comment:
```suggestion
Whenever a new image is built and released to AWS, all the previous serials for that image are deprecated except for the last 3. A deprecated image is not visible on the AWS console, but it can still be found and launched from the `AWS CLI`_ using its AMI ID. You can also view the image details using the AWS CLI `describe-images`_ command by including the ``--include-deprecated`` flag.
```